cfme/common/provider.py

```diff
@@ -167,6 +167,14 @@ def get_yaml_data(self):
             raise ProviderHasNoKey(
                 'Provider {} has no key, so cannot get yaml data'.format(self.name))
 
+    def has_nodes(self):
```

Comment:
I'm not sure if this is the logic but I would implement it something like (didn't try it - only assumption):
def has_nodes(self):
    if not self.summary.relationships.hosts.value:
        return False
    return self.summary.relationships.nodes.value > 0